pkg/runner/cluster_k8s.go

```diff
@@ -976,9 +968,10 @@ func (c *ClusterK8sRunner) checkClusterResources(ow *rpc.OutputWriter, groups []
 
 	// all worker nodes are the same, so just take allocatable CPU from the first
 	item := res.Items[0].Status.Allocatable["cpu"]
-	nodeCPUs, _ := item.AsInt64()
+	nodeCPUs := item.ToDec().Value()
```

Comment:
Do you know why fast conversion is not available on the EKS?

> For the test cluster, the final tally comes down to this:
Needed: 0.200000, vs available: 31.600000
Since we have a total of 32vCPUs (2 nodes, 16vCPUs each), the math checks out.

What does this specific function call relate to all this? I feel like I'm missing some mental bridge here.

Reply:
> No idea; is there a reason we really need the fast conversion?

This is not critical so feel free to attend to other matters. If we change the code, I usually like to know why.

As for the rest, now I do get where these numbers are coming from. Thank you for the explanation! It makes total sense 👍